README.md

```diff
@@ -16,8 +16,13 @@ specification](#custom-tuning-specification) is passed to all
 daemons running in the cluster in the format that the daemons understand.
 The daemons run on all nodes in the cluster, one per node.
 
+Node-level settings applied by the containerized tuned are rolled back on
+an event that triggers a profile change or when the containerized tuned
```

Comment:
I guess I am little confused on what the settings are rolled back to ?

Reply:
All the settings changed by a profile (e.g. sysctls) are rolled back upon application of a new profile.  Say profile A sets `net.ipv4.ip_local_port_range="1024 65535"`.  Before profile A is set though, tuned daemon looks at `net.ipv4.ip_local_port_range` and sees the default value `32768 60999`.  It remembers that value and upon a change to profile B `net.ipv4.ip_local_port_range` is unapplied/rolled-back to `32768 60999`.